bode/bode/models/task_actions.py

```diff
@@ -33,20 +33,20 @@ def is_interchangable_relation(relation):
     task.title = task_data["title"]
     task.description = task_data["description"]
     task.due_date = task_data["due_date"]
-    task.is_done = task_data["is_done"]
+    task.status = task_data["status"]
```

Comment:
I think the process of populating `Task` object using dictionary can be done like so:
`for key, value in task_data.items():`
`   setattr(task, key, value)`
What do you think?